app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/MappingWizardClustersStep.js

```diff
@@ -12,6 +18,13 @@ class MappingWizardClustersStep extends React.Component {
       mappings: [], // eslint-disable-line react/no-unused-state
       selectedMapping: null // eslint-disable-line react/no-unused-state
     };
+
+    this.selectSourceCluster = this.selectSourceCluster.bind(this);
+    this.selectTargetCluster = this.selectTargetCluster.bind(this);
+    this.addMapping = this.addMapping.bind(this);
```

Comment:
we have bind helpers for these now... just import `bindMethods` from patternfly-react

Reply:
example: https://github.com/patternfly/patternfly-react/blob/master/src/components/Table/__mocks__/mockClientPaginationTable.js#L60

Reply:
Whoops!  Ok, I'll get this switched out!